src/tests/Common/Coreclr.TestWrapper/Coreclr.TestWrapper.csproj

```diff
@@ -4,7 +4,10 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <NuGetTargetMoniker>$(NetCoreAppCurrentToolTargetFrameworkMoniker)</NuGetTargetMoniker>
     <NuGetTargetMonikerShort>$(NetCoreAppToolCurrent)</NuGetTargetMonikerShort>
-    <RunAnalyzers>false</RunAnalyzers>
+    <Nullable>enable</Nullable>
+    <RunAnalyzers>true</RunAnalyzers>
+    <EnableAotAnalyzer>true</EnableAotAnalyzer>
+    <EnableTrimAnalyzer>true</EnableTrimAnalyzer>
```

Comment:
For my own education, why were these 3 `*Analyzer` properties needed? How do they affect the output?

Reply:
These enable all the trim/AOT analyzers. They're otherwise disabled by default because the runtime tests aren't AOT-compatible usually.